README.md

```diff
@@ -45,6 +45,7 @@ If you have any questions, suggestions let's have it over [Issues](https://githu
 
 - 📦 Safe Storage
   * [Cryptography of sharedpreferences](https://developer.android.com/reference/androidx/security/crypto/EncryptedSharedPreferences)
+  * [DataStore(modern data storage solution)](https://developer.android.com/topic/libraries/architecture/datastore)
```

Comment:
Data Store can be used as encrypted but I believe that the best solution is Data Store + CryptoManager or any kind of encryption before store it inside the data store to be secure.

```suggestion
  * [DataStore + Cipher (to secure the data)
  (modern data storage solution)](https://developer.android.com/topic/libraries/architecture/datastore)
```

Reply:
This suggestion is because the target of this repo, is to share content related with security, and ways to work more securely.

Reply:
it depends on  what library u want to encrypt it with, but u r absolutely right